modules/create-a-custom-access-scope.adoc

```diff
@@ -46,3 +46,16 @@ Access to a specific namespace gives access to the following information within
 ====
 . If you want to allow access to clusters and namespaces based on labels, click *Add label selector* under the *Label selection rules* section. Then click *Add rule* to specify *Key* and *Value* pairs for the label selector. You can specify labels for clusters and namespaces.
 . Click *Save*.
+
```

Comment:
I would remove line 49-53 and make "Scale considerations" into a heading like this:

```
[id="scale-considerations_{context}"]
== Scale considerations
```